CONTRIBUTING.md

```diff
@@ -10,4 +10,5 @@ This is a pure rust project, so only `cargo` is needed.
 - `cargo fmt` to format the current code.
```

Comment:
maybe we need to give a hint about using "cargo +nightly fmt" here?

Reply:
done